sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentServiceResponse.java

```diff
@@ -3,6 +3,7 @@
 
 package com.azure.cosmos.implementation;
 
+import com.azure.cosmos.Permission;
```

Comment:
undo?

Reply:
this is required. as permission is moved from one package to another and hence not accessible in the old package.